packages/mobile/components/Drawer.svelte

```diff
@@ -46,18 +53,21 @@
     />
     <panel
         on:touchstart={onTouchStart}
-        in:fly={{ y: 100, duration: 300 }}
-        out:fly={{ y: 100, duration: 200 }}
+        in:fly={{ ...directon, duration: 300 }}
+        out:fly={{ ...directon, duration: 200 }}
         bind:clientHeight={panelHeight}
+        bind:clientWidth={panelWidth}
         class:moving
         class="py-6 px-5 fixed w-full flex flex-col flex-auto {fullScreen
             ? 'h-screen'
-            : ''} z-10 bg-white dark:bg-gray-800 rounded-t-2xl"
-        style="bottom: {panelBottom}px;"
+            : ''} z-10 bg-white dark:bg-gray-800 {enterFromSide ? '' : 'rounded-t-2xl'}"
+        style={enterFromSide ? `left: ${position}px;` : `bottom: ${position}px;`}
     >
-        <decorator
-            class="absolute top-2 left-1/2 transform -translate-x-1/2 w-12 h-1 bg-gray-300 dark:bg-gray-700 rounded"
-        />
+        {#if enterFromSide === false}
+            <decorator
+                class="absolute top-2 left-1/2 transform -translate-x-1/2 w-12 h-1 bg-gray-300 dark:bg-gray-700 rounded"
```

Comment:
we also have this line in the CSS, shouldnt we maybe also add the left transition? `transition: bottom 0.2s ease;`